views/dashboard-application.hbs

```diff
@@ -105,15 +107,11 @@
                                 <option {{#if user.schoolOther}} selected{{/if}}>Other</option>
                             </select>
                         </div>
-                        <div class='form-group col-md-5'>
-                            <label>If Other, please specify.</label>
-                            <input class='form-control' type='text' name='school_other' placeholder='Other' value='{{user.school_other}}'>
-                        </div>
-                        <div class='form-group col-md-5'>
```

Comment:
We do still want this question. If the applicant isn't from a UC, we'd like to know where they're from. I know design doesn't have it in the Figma, but they weren't told what fields we need on the application